clang_9/Dockerfile

```diff
@@ -90,9 +81,35 @@ RUN dpkg --add-architecture i386 \
     && update-alternatives --install /usr/bin/pip pip /opt/pyenv/shims/pip 100 \
     && update-alternatives --install /usr/bin/pip3 pip3 /opt/pyenv/shims/pip3 100
 
+
+FROM basics AS building_layer 
+
+RUN mkdir conan_packages \
+    && cd conan_packages \
+    # Temporary dependencies for (potential) boostrapping purposes 
+    && wget --no-check-certificate --quiet https://cmake.org/files/v${CMAKE_VERSION_MAJOR_MINOR}/cmake-${CMAKE_VERSION_FULL}.tar.gz \
+    && tar -xzf cmake-${CMAKE_VERSION_FULL}.tar.gz \
+    && cd cmake-${CMAKE_VERSION_FULL} \
+    && ./bootstrap > /dev/null \
+    && make -s -j`nproc` \
+    && make -s install > /dev/null \
+    && cd - \
+    && rm -rf cmake-* \
+    # Install Packages which we want to deploy
+    # Install CMake + move license files to specific licenses/cmake subdirectory for deploy
+    && conan install cmake/${CMAKE_VERSION_FULL}@ --build missing -g deploy \
+    && cd cmake/licenses \
+    && mkdir cmake \ 
+    && ls | grep -v cmake | xargs mv -t cmake
```

Comment:
I don't completely understand everything you do here, but would it also be possible to
use a `conanfile.py` script with a `deploy` method here?
When/if multiple conan packages will get pre-installed, then it would be a matter of updating the conanfile, instead of all dockerfiles.

Reply:
The CMake version is already in every `Dockerfile`. 

And in the feature we will have much less `Dockerfile`s to maintain either due to something like #204 or by using `ARGS` for values which differ between the `Dockerfile`s, or both. The second thing is something I would like to check out next, as mass editing of so many  `Dockerfile`s continuously lead to errors.

Either way, currently there are differences. For instance, in the GCC 4.8 image we always have to rebuild CMake, as the package is build in the GCC 4.9 image with a newer GLIBC/GLIBCXX versions and can not run on the GCC 4.8 image.

Newer CMake versions might also not compile anymore in old compiler. And thinking ahead, I believe newer Python versions already have issues with too old distro versions IIRC. 🤔  

All in all, I don't think a single `conanfile.py` would really simplify much right now, but might be worth considering in the future again